src/Webhooks/Webhooks.csproj

```diff
@@ -45,8 +45,20 @@
     <DelaySign>false</DelaySign>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="!GameSource">
+      <HintPath>..\..\..\..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\BROKE PROTOCOL\Plugins\!GameSource.dll</HintPath>
+    </Reference>
+    <Reference Include="!LifeSource">
+      <HintPath>..\..\..\..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\BROKE PROTOCOL\Plugins\!LifeSource.dll</HintPath>
+    </Reference>
+    <Reference Include="Assembly-CSharp">
+      <HintPath>..\..\..\..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\BROKE PROTOCOL\BrokeProtocol_Data\Managed\Assembly-CSharp.dll</HintPath>
+    </Reference>
+    <Reference Include="AstarPathfindingProject">
+      <HintPath>..\..\..\..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\BROKE PROTOCOL\BrokeProtocol_Data\Managed\AstarPathfindingProject.dll</HintPath>
+    </Reference>
```

Comment:
Can we use the `BPDIR` environment variable (DM me if you don't know what that means)?

Reply:
I didn't know what that is ._.